code/datums/status_effects/neutral.dm

```diff
@@ -342,3 +342,186 @@
 	tick_interval = INFINITY
 	status_type = STATUS_EFFECT_REFRESH
 	alert_type = null
+
+/datum/status_effect/eigenstasium
+	id = "eigenstasium"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = null
+	processing_speed = STATUS_EFFECT_NORMAL_PROCESS
+	///So we know what cycle we're in during the status
+	var/current_cycle = -250 //Consider it your stability
+	///The addiction looper for addiction stage 3
+	var/phase_3_cycle = -0 //start off delayed
+	///Your clone from another reality
+	var/mob/living/carbon/alt_clone = null
+	///If we display the stabilised message or not
+	var/stable_message = FALSE
+
+/datum/status_effect/eigenstasium/Destroy()
+	UnregisterSignal(alt_clone, COMSIG_PARENT_QDELETING)
+	QDEL_NULL(alt_clone)
+	return ..()
+
+/datum/status_effect/eigenstasium/tick()
+	. = ..()
+	//This stuff runs every cycle
+	if(prob(5))
+		do_sparks(5, FALSE, owner)
+
+	//If we have a reagent that blocks the effects
+	var/block_effects = FALSE
+	if(owner.has_reagent(/datum/reagent/bluespace))
+		current_cycle = max(-250, (current_cycle - 7.5)) //cap to -250
```

Comment:
Make these defines too, plus in the other place `-250` is used--you don't want to have to update two places at once.